eve/main.yml

```diff
@@ -512,6 +512,34 @@ stages:
           haltOnFailure: True
       - SetPropertyFromCommand: *set_version_prop
       - SetPropertyFromCommand: *set_short_version_prop
+      - TriggerStages:
+          name: Trigger post-merge lifecycle and solutions stages simultaneously
+          stage_names:
+            - post-merge-lifecycle
+            - post-merge-solutions
+          haltOnFailure: true
+      - ShellCommand: *add_final_status_artifact_success
+      - Upload: *upload_final_status_artifact
+      - TriggerStages:
+          name: Trigger TestRail objects creation and results upload
+          stage_names:
+            - create-upload-testrail-objects
+          alwaysRun: True
+
+  post-merge-solutions:
+    worker:
+      type: local
+    steps:
+      - TriggerStages:
+          name: Trigger build of Example Solution archive
+          stage_names:
+            - build-example-solution
+          haltOnFailure: true
```

Comment:
It's a stage triggering a single stage, it seems pretty useless, but it's done this way for the code to be similar between 2.4 and 2.5+, so we don't have conflicts if we change something in 2.4.
Because, in later versions, we also trigger Solutions frameworks tests right after this stage.

Reply:
Not yet though, right (at least, can't see any difference in integration branches)?

Reply:
Now it's done: https://github.com/scality/metalk8s/blob/w/2.6/improvement/rework-post-merge-stages/eve/main.yml